de.tudarmstadt.ukp.wikipedia.timemachine/src/main/java/de/tudarmstadt/ukp/wikipedia/timemachine/dump/version/DumpVersionTroveIntKey.java

```diff
@@ -45,29 +45,29 @@ public class DumpVersionTroveIntKey extends AbstractDumpVersion {
 	 * after revision parsing the map will be erased and the keys sorted in the
 	 * array list
 	 */
-	private TIntHashSet pageIdRevList;
+	private Set<Integer> pageIdRevList;
```

Comment:
Wouldn't using something like FastUtil be reasonable here?

Reply:
Switched to fastutil-core